src/main/splitting/tasks_tests_handler.py

```diff
@@ -0,0 +1,290 @@
+        return False
+
+
+def __compile_program(call_args: list):
+    try:
+        return call(call_args) == 0
+    except:
+        return False
+
+
+def __get_args_for_running_program(language: str, source_file_name: str):
+    if language == LANGUAGE.JAVA.value:
+        running_args = ['java', '-cp', __get_source_folder(), source_file_name]
+
+    elif language == LANGUAGE.CPP.value:
+        running_args = [__get_compiled_file(source_file_name) + '.out']
+    elif language == LANGUAGE.KOTLIN.value:
+        running_args = ['java', '-jar', __get_compiled_file(source_file_name) + '.jar']
+    else:
+        raise ValueError('Language is not defined')
+    return running_args
+
+
+def __get_args_for_compiling_program(language: LANGUAGE, source_file_name: str):
+    compiled_file = __get_compiled_file(source_file_name)
+    extension = get_extension_by_language(language)
+
+    if language == LANGUAGE.JAVA.value:
+        compiled_file_path = compiled_file + '.' + extension
+        call_args = ['javac', compiled_file_path]
+    elif language == LANGUAGE.CPP.value:
+        compiled_file_path = compiled_file + '.out'
+        call_args = ['g++', '-o', compiled_file_path, compiled_file + '.' + extension]
+    elif language == LANGUAGE.KOTLIN.value:
+        compiled_file_path = compiled_file + '.jar'
+        call_args = ['kotlinc', compiled_file + '.' + extension, '-include-runtime', '-d',
+                     compiled_file_path]
+    else:
+        raise ValueError('Language is not defined')
+
+    return call_args
+
+
+def __create_source_code_file(source_code: str, language=LANGUAGE.PYTHON.value,
+                              source_file_name=SOURCE_OBJECT_NAME):
+    if language == LANGUAGE.JAVA.value:
+        source_file_name = __get_java_class(source_code)
+    create_file(source_code, get_extension_by_language(language), __get_compiled_file(source_file_name))
+    return source_file_name
+
+
+def create_in_and_out_dict(tasks: list):
+    in_and_out_files_dict = {}
+    for task in tasks:
+        files = next(os.walk(TASKS_TESTS_PATH + task))[2]
+        in_and_out_files_dict[task] = __get_in_and_out_files(files)
+
+    return in_and_out_files_dict
+
+
+def is_python_file_correct(filename: str):
+    source = open(filename, 'r').read() + '\n'
+    if check_python_file_by_mypy(filename):
+        try:
+            code = compile(source, filename, 'exec')
+            signal.alarm(consts.MAX_SECONDS_TO_WAIT_TEST)
+            eval(code, {})
+            return True
+        except TimeoutException:
+            # It means that eval didn't throw any exceptions, but time is out for example because of input waiting
+            log.info("Time is out")
+            return True
+        except:
+            return False
+    return False
+
+
+def check_python_file_by_mypy(file_name: str):
+    call_args = ['mypy', file_name]
+    is_valid = __compile_program(call_args)
+    return is_valid
```

Comment:
Delete the additional variable is_valid. We should use: return __compile_program(call_args)